### DISCUSSION TITLE
Thank you for this project, _really_ easy to get going

### DISCUSSION THREAD

**Discussion by itamarst** [Pagefind/pagefind#79] [Show and tell]
You can see results at https://pythonspeed.com/

I really appreciate the low bandwidth involved, too.

**Comment by bglw** [comment]
Awesome! Glad to hear ☺️

**Comment by dprothero** [comment]
Huge +1 to this.

**Comment by codeflyer47** [comment]
Agreed. This search is the best. Thank you so much! ✨🤩✨
